_data/people/rct225.yml

```
@@ -0,0 +1,8 @@
+name: Robert Tuck
+shortname: rct225
+title: <Can be blank - will show a title under your picture>
```

```suggestion
title:
```

Please put a title here, or leave it blank. Thanks!

---

I'll add it